extensions-core/druid-kerberos/src/main/java/io/druid/security/kerberos/KerberosAuthenticator.java

```diff
@@ -236,6 +230,12 @@ public void doFilter(
 
         String path = ((HttpServletRequest) request).getRequestURI();
         if (isExcluded(path)) {
+          // For excluded paths that does NOT need to be authenticated.
```

Comment:
If there's an authorizer configured + the Kerberos authenticator, this won't be enough to skip security checks on a request (the authorizer would check what permissions the "anonymous" user would have and deny the request unless the user has configured appropriate permissions for the "anonymous" user)

To support `excludedPaths`, there's a similar concept with "unsecured paths" in the various service initialization files (e.g., UNSECURED_PATHS in CoordinatorJettyServerInitializer), one option could be to extend that to allow user-specified whitelist entries (i.e., make `excludedPaths` a general concept applied across all authenticators, and remove `excludedPaths` as a Kerberos-specific config). 

I think `excludedPaths` makes more sense as a system-wide property than an authenticator-specific property, I'll suggest taking this out of this PR and one of us could do follow-on PR to make it a general config.

Reply:
@jon-wei: sure, we can make it a general config.
Will raise a separate PR for that.

Reply:
@jon-wei, created  https://github.com/druid-io/druid/pull/5614